kroxylicious-sample/pom.xml

```diff
@@ -0,0 +1,77 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright Kroxylicious Authors.
+
+    Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+
+-->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>io.kroxylicious</groupId>
+        <artifactId>kroxylicious-parent</artifactId>
```

Comment:
Maybe the sample should stand alone without inheriting the kroxylicious-parent configuration so that it can be more easily copied (imagining a user copying the `kroxylicious-sample` directory into a new repository to get started). The downside being it might need additional steps at release time to update versions.

Reply:
I wondered about that too, I think thats a refactoring for another day.

I think a template repo or a maven archetype might be better options for the new filter bootstrap question.